include/simdjson/document.h

```diff
@@ -339,7 +345,8 @@ class document {
    *
    * The parser will allocate capacity as needed.
    */
-  document() noexcept = default;
+  ///Os3 2
+  document() noexcept :tape(), string_buf() {};
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR.

Reply:
done